src/DonationForms/Properties/FormSettings.php

```diff
@@ -234,10 +234,9 @@ public static function fromArray(array $array): self
      */
     public static function fromJson(string $json): self
     {
-        $self = new self();
-        $array = json_decode($json, true);
-
-        return $self::fromArray($array);
```

Comment:
`FormSettings::fromArray()` is a static method and creates its own instance of the `FormSettings` object.